### ISSUE TITLE
Issue with "Array.prototype.fill()": (short summary here please)

### ISSUE THREAD

**Issue by domleonard** [mdn/content#7371]
MDN URL: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/fill

#### What information was incorrect, unhelpful, or incomplete?

Description of the `end` parameter was incomplete, leading to extra debugging time.

#### Specific section or headline?

Description

#### What did you expect to see?

(in addition to existing description)

The `end` parameter is treated as `array.length` if it is greater than `array.length` - you can't lengthen an array using the `fill` method without setting `array.length` to the required length first.

#### Did you test this? If so, how?

Yes.  By evalaluating

``` js
let hash = [];
hash.length = 256;  // omit and the array is not modified
console.log(hash.fill(0, 0, 256) );
```

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/javascript/reference/global_objects/array/fill`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/fill
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/javascript/reference/global_objects/array/fill/index.md
* Last commit: https://github.com/mdn/content/commit/cf607d68522cd35ee7670782d3ee3a361eaef2e4
* Document last modified: 2021-07-20T18:47:57.000Z

</details>

**Comment by Anurella** [comment]
@domleonard 

Array.fill is used to change elements in array, to use it on a an empty array would not work because there is nothing to change, the best way is to assign empty slots to the array as you have done in your debug example, before using the fill method. 

A note can be added to the article about using the fill method on empty arrays just after the example

**Comment by Anurella** [comment]
@sideshowbarker I can work on updating the article

**Comment by Ryuno-Ki** [comment]
Please do, @Anurella!

**Comment by Anurella** [comment]
@Ryuno-Ki Made the changes and pushed but cannot create a new pull request until my previous request has been closed. 

Thank you

**Comment by Ryuno-Ki** [comment]
Ah, yeah, the thing with mixed PRs, right?
Hopefully your PRs get merged soon, so that you get unblocked.
